### PR TITLE
[zh-cn] update types link in StorageEvent

### PR DESCRIPTION
**Pull request by sechi747**

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
Hi @sechi747. Could you convert this table into [definition list](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E5%AE%9A%E4%B9%89%E5%88%97%E8%A1%A8), just as it is in [en-US](https://github.com/mdn/content/blob/fa93ec954ca369092902573e76ede0009d15f238/files/en-us/web/api/storageevent/index.md?plain=1#L25-L41).

If you have any problem with this, please let me know :)

**Comment by sechi747** [comment]
> Hi @sechi747. Could you convert this table into [definition list](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E5%AE%9A%E4%B9%89%E5%88%97%E8%A1%A8), just as it is in [en-US](https://github.com/mdn/content/blob/fa93ec954ca369092902573e76ede0009d15f238/files/en-us/web/api/storageevent/index.md?plain=1#L25-L41).
> 
> If you have any problem with this, please let me know :)

Have done, review it please.

File: files/zh-cn/web/api/storageevent/index.md [hunk 1]
```diff
@@ -30,13 +30,18 @@ void initStorageEvent(
...
 ## 属性
 
-| 属性名        | 类型                         | 描述                                                                                                       |
-| ------------- | ---------------------------- | ---------------------------------------------------------------------------------------------------------- |
-| `key`         | [`DOMString`](/zh-CN/DOMString) | 该属性代表被修改的键值。当被 clear() 方法清除之后该属性值为 null。**（只读）**                             |
-| `newValue`    | [`DOMString`](/zh-CN/DOMString) | 该属性代表修改后的新值。当被 clear() 方法清理后或者该键值对被移除，`newValue` 的值为 `null` 。**（只读）** |
-| `oldValue`    | [`DOMString`](/zh-CN/DOMString) | 该属性代表修改前的原值。在设置新键值对时由于没有原始值，该属性值为 `null`。**（只读）**                    |
-| `storageArea` | `nsIDOMStorage`              | 被操作的 storage 对象。**（只读）**                                                                        |
-| `url`         | [`DOMString`](/zh-CN/DOMString) | key 发生改变的对象所在文档的 URL 地址。**（只读）**                                                        |
+除了下面列出的属性以外，`StorageEvent` 还继承了 {{domxref("Event")}} 的属性。
```
Comment (yin1999) [inline files/zh-cn/web/api/storageevent/index.md:33]:
```suggestion
_除了下面列出的属性以外，`StorageEvent` 还继承了父接口 {{domxref("Event")}} 的属性。_
```

File: files/zh-cn/web/api/storageevent/index.md [hunk 2]
```diff
@@ -30,13 +30,18 @@ void initStorageEvent(
...
+
+- {{domxref("StorageEvent.key", "key")}} {{ReadOnlyInline}}
+  - : 被修改的键名。当被 `clear()` 方法清除后该属性值为 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null)。
+- {{domxref("StorageEvent.newValue", "newValue")}} {{ReadOnlyInline}}
+  - : 修改后的新值。当被 `clear()` 方法清除或者该键值对被移除时，`newValue` 的值为 `null` 。
```
Comment (yin1999) [inline files/zh-cn/web/api/storageevent/index.md:38]:
```suggestion
  - : 修改后的新值。当被 `clear()` 方法清除或者从存储中移除该键值对时，`newValue` 的值为 `null` 。
```

File: files/zh-cn/web/api/storageevent/index.md [hunk 3]
```diff
@@ -30,13 +30,18 @@ void initStorageEvent(
...
+- {{domxref("StorageEvent.oldValue", "oldValue")}} {{ReadOnlyInline}}
+  - : 修改前的旧值。在设置新键值对时由于没有原始值，该属性值为 `null`。
```
Comment (yin1999) [inline files/zh-cn/web/api/storageevent/index.md:40]:
```suggestion
  - : 该键（`key`）在修改前的旧值。在添加新键值对时由于没有原始值，该属性值为 `null`。
```

**Review by yin1999 — COMMENTED** [review]
Thanks. Just some suggestions

**Comment by sechi747** [comment]
> Thanks. Just some suggestions

Thanks, I learned a lot.😄

**Review by yin1999 — APPROVED** [review]
Thanks :tada:

**Pull request merged**